src/app/pages/listing/listing-edit-upload.jsx

```diff
@@ -457,24 +320,150 @@ function ChplListingEditUploadPage({ id }) {
                 </CardContent>
               </Card>
             </ListingContext.Provider>
-          </div>
-          <div>
-            { newListing
-              && (
-                <ListingContext.Provider value={newListingState}>
-                  <Card>
-                    <Box className={classes.sectionHeader}>
-                      <Typography className={classes.sectionHeaderText} variant="h2">Additional Information</Typography>
-                    </Box>
-                    <CardContent>
-                      <ChplAdditionalInformation
-                        listing={newListing}
-                      />
-                    </CardContent>
-                  </Card>
-                </ListingContext.Provider>
+          </Box>
+          </Box>
+          <Box>{/* Upload Container */}
+            <Typography gutterBottom component={"h3"} style={{ fontWeight: '600' }} variant='h4'>Updated Listing</Typography>
+            <div>
+              {!newListing ? (
+                <Box className={classes.placeholderContainer}>
+                  <HelpOutlineIcon fontSize="large" color="primary" />
+                  <Typography>Upload a file above to display your new listing.</Typography>
+                </Box>
+              ) : (
+                <>
+                  {newListing && (
```

Comment:
The first line in this block starts a ternary check. If the check is true, it will show the first element (the temporary text stuff). If it's false, it will show the stuff after the `:`. The last line in this block is re-checking that the `newListing` exists, which it _absolutely must_ for it to even get into this branch of the ternary